proprietary/design-tokens/src/component/utrecht/combobox.tokens.json

```diff
@@ -0,0 +1,37 @@
+{
+  "utrecht": {
+    "combobox": {
+      "max-inline-size": { "value": "448px" },
```

Comment:
Kunnen we hiervoor ook gebruik maken van `var(--utrecht-form-input-max-inline-size)` als fallback?